functions/system.bash

```diff
@@ -314,20 +315,20 @@ Finally, all common serial ports can be made accessible to the openHAB java virt
   fi
 
   if [[ $selection == *"2"* ]]; then
-    if is_pithree || is_pithreeplus; then
+    if is_pithree || is_pithreeplus || is_pifour; then
       #cond_redirect systemctl stop hciuart &>/dev/null
       #cond_redirect systemctl disable hciuart &>/dev/null
-      cond_echo "Adding 'dtoverlay=pi3-miniuart-bt' to /boot/config.txt (RPi3)"
-      if ! grep -q "dtoverlay=pi3-miniuart-bt" /boot/config.txt; then
-        echo "dtoverlay=pi3-miniuart-bt" >> /boot/config.txt
+      cond_echo "Adding 'dtoverlay=miniuart-bt' to /boot/config.txt (RPi3/4)"
+      if ! grep -q "dtoverlay=miniuart-bt" /boot/config.txt; then
```

Comment:
Shall we open a new issue for the BT stuff?
Looking at the docs, this cannot work properly without setting cpu_freq / cpu_freq_max.
So it seems we are "disabling" BT ;-)

I think the intention here was to prepare the serial port for a HAT, switching to the alternate BT possibility.
I don't know it this is widely used. I would prefer to have - somewhere - a clear way of disabling all wireless stuff.

Reply:
Yes that's the intention of this code section: to enable the serial interface to make control work  for HATs.
It is also my understanding we disable BT here as a side effect.
But that isn't new, it has been like that ever on Pi3, Pi4.
Feel free to add separate issues/features to investigate disabling or enabling alternate BT possibilities.
HATs to work are the priority here. I fear disabling BT could affect serial via miniuart, too.
I don't have HW to test with so I do not dare to mess around with that. All I adjusted was the option name (which has the "pi3" removed because it now applies to Pi4 as well).